utils/safePin.ts

```diff
@@ -11,7 +11,7 @@ export async function pinImageSafe(
     return IPFS_KODADOT_IMAGE_PLACEHOLDER
   }
 
-  return pinFileToIPFS(file, token)
+  return await pinFileToIPFS(file, token)
```

Comment:
Useless await?

Reply:
since it's async function I don't think so? plus any use of `pinFileToIPFS` in codebase use async/await